R/navs-legacy.R

```diff
@@ -234,6 +256,25 @@ navbar_options <- function(
   )
 }
 
+navbar_options_apply_attribs <- function(navbar, navbar_options = NULL) {
+  if (is.null(navbar_options[["attribs"]])) {
+    return(navbar)
+  }
+
+  attribs <- navbar_options[["attribs"]]
+  navbar[[1]] <- rlang::exec(tagAppendAttributes, navbar[[1]], !!!attribs)
+
+  if ("data-bs-theme" %in% names(attribs)) {
+    # If you're setting this attribute directly, you know more about what you're
+    # doing than we do (we handle it for users via `type`). Also: the call to
+    # tagAppendAttributes ensures that `navbar[[1]]` is a tag object and has the
+    # attribs field.
+    navbar[[1]][["attribs"]][["data-bs-theme"]] <- attribs[["data-bs-theme"]]
+  }
```

Comment:
I must be missing something here, but wouldn't `navbar[[1]]` be the same if this portion was removed?

Reply:
No, `tagAppendAttributes()` concatenates attributes, here we want to overwrite it.

Reply:
Ahh, right, ok